src/main/java/com/rakovets/course/java/core/practice/oop/classesandobjects/Time.java

```diff
@@ -0,0 +1,46 @@
+package com.rakovets.course.java.core.practice.oop.classesandobjects;
+
+public class Time {
+    private int hours;
+    private int minutes;
+    private int seconds;
+
+    public Time(int totalSeconds) {
+        this.seconds = totalSeconds;
+    }
```

Comment:
Why does specification contain 2 variables for one business value?
`seconds` != `totalSeconds`